Resources/Prototypes/Entities/Constructible/Power/saltern_power.yml

```
@@ -34,6 +34,7 @@
 
 - type: entity
   id: SalternApc
+  description: A control terminal for the area electrical systems.
```

Maybe `area's` here instead?
```suggestion
  description: A control terminal for the area's electrical systems.
```

---

Sure!  I literally just copied it from the Aurorastation code, so *shrug*.  I can see a grammatical case for the former, but whatever works : D